content/docs/digging_deeper/transmit.md

```diff
@@ -0,0 +1,373 @@
+  pingInterval: false,
+  transport: null
+})
+```
+
+<dl>
+
+<dt>
+
+pingInterval
+
+</dt>
+
+<dd>
+
+The interval used to send ping messages to the client. The value is in milliseconds or using a string `Duration` format (i.e: `10s`). Set to `false` to disable ping messages.
+
+</dd>
+
+<dt>
+
+transport
+
+</dt>
+
+<dd>
+
+Transmit supports syncing events across multiple servers or instances. You can enable the feature by referencing the wanted transport layer (only `redis` is supported for now). Set to `null` to disable syncing.
+
+```ts
+import env from '#start/env'
+import { defineConfig } from '@adonisjs/transmit'
+import { redis } from '@adonisjs/transmit/transports'
+
+export default defineConfig({
+  transport: {
+    driver: redis({
+      host: env.get('REDIS_HOST'),
+      port: env.get('REDIS_PORT'),
+      password: env.get('REDIS_PASSWORD'),
+    })
+  }
+})
+```
+
+:::note
+Ensure you have `ioredis` installed when using the `redis` transport.
+:::
+
+</dd>
+
+</dl>
+
+## Channels
+
+Channels are used to group events. For example, you can have a channel for notifications, another for chat messages, and so on.
+
+### Channel Names
+
+Channel names are used to identify the channel. They are case-sensitive and must be a string. You cannot use any special characters or spaces in the channel name except `/`. The following are some examples of valid channel names:
+
+```ts
+transmit.broadcast('global', { message: 'Hello' })
+transmit.broadcast('chats/1/messages', { message: 'Hello' })
+transmit.broadcast('users/1', { message: 'Hello' })
+```
+
+:::tip
+Channel names use the same syntax as route in AdonisJS.
+:::
+
+### Channel Authorization
+
+You can authorize or reject a connection to a channel using the `authorizeChannel` method. The method receives the channel name and the `HttpContext`. It must return a boolean value.
+
+```ts
+// title: start/transmit.ts
+
+import Chat from '#models/chat'
+import type { HttpContext } from '@adonisjs/core/http'
+
+transmit.authorizeChannel<{ id: string }>('users/:id', (ctx: HttpContext, { id }) => {
+  return ctx.auth.user?.id === +id
+})
+
+transmit.authorizeChannel<{ id: string }>('chats/:id/messages', async (ctx: HttpContext, { id }) => {
+  const chat = await Chat.findOrFail(+id)
+  
+  return ctx.bouncer.allows('accessChat', chat)
+})
+```
+
+## Broadcasting Events
+
+You can broadcast events to a channel using the `broadcast` method. The method receives the channel name and the data to send.
+
+```ts
+transmit.broadcast('global', { message: 'Hello' })
+```
+
+You can also broadcast events to any channel except one using the `broadcastExcept` method. The method receives the channel name, the data to send, and the UID you want to ignore.
+
+```ts
+transmit.broadcastExcept('global', { message: 'Hello' }, 'uid-of-sender')
+```
+
+## Listening for Events
+
+You can listen for events on the client-side using the `@adonisjs/transmit-client` package. The package provides a `Transmit` class.
```

Comment:
Build on top of the SSE object (by default) with a link to MDN?

Reply:
The information is already available bellow when we are presenting the different parameters of the client.

Do you believe it still makes sens to add it here?